assets/js/min/main.min.js

```diff
@@ -1,3 +1,3 @@
hTemplate),this.$el.append(this.template),this.model.each(function(b){a.renderItem(b)}),this},search:function(a,b){var c=this,d={__nonce:this.nonce,ap_ajax_action:"load_filter_"+this.filter,search:a,filter:this.filter};AnsPress.showLoading(b),AnsPress.ajax({data:d,success:function(a){if(AnsPress.hideLoading(b),a.success){for(c.nonce=a.nonce;model=c.model.first();)model.destroy();c.model.add(a.items)}}})},searchInput:function(b){var c=this;clearTimeout(this.searchTO),this.searchTO=setTimeout(function(){c.search(a(b.target).val(),b.target)},600)},added:function(a){this.renderItem(a)}}),AnsPress.views.List=Backbone.View.extend({el:"#ap-filters",initialize:function(){},events:{"click [ap-filter]:not(.loaded)":"loadFilter","click #ap-filter-reset":"resetFilter"},loadFilter:function(b){b.preventDefault();AnsPress.showLoading(b.currentTarget);var c=a.parseJSON(a(b.currentTarget).attr("apquery"));c.ap_ajax_action="load_filter_"+c.filter,AnsPress.ajax({data:c,success:function(d){AnsPress.hideLoading(b.currentTarget),a(b.currentTarget).addClass("loaded");var e=new AnsPress.collections.Filters(d.items),f=new AnsPress.views.Filters({model:e,multiple:d.multiple,filter:c.filter,nonce:d.nonce});a(b.currentTarget).after(f.render().$el)}})},resetFilter:function(b){a('#ap-filters input[type="hidden"]').remove(),a('#ap-filters input[type="checkbox"]').prop("checked",!1)}}),a(document).ready(function(){new AnsPress.views.List})}(jQuery),function(a){AnsPress.models.Notification=Backbone.Model.extend({idAttribute:"ID",defaults:{ID:"",verb:"",verb_label:"",icon:"",avatar:"",hide_actor:"",actor:"",ref_title:"",ref_type:"",points:"",date:"",permalink:"",seen:""}}),AnsPress.collections.Notifications=Backbone.Collection.extend({model:AnsPress.models.Notification}),AnsPress.views.Notification=Backbone.View.extend({id:function(){return"noti-"+this.model.id},template:'<div class="noti-item clearfix {{seen==1 ? \'seen\' : \'unseen\'}}"><# if(ref_type === \'reputation\') { #>  <div class="ap-noti-rep<# if(points < 1) { #> negative<# } #>">{{points}}</div><# } else if(hide_actor) { #><div class="ap-noti-icon {{icon}}"></div><# } else { #><div class="ap-noti-avatar">{{{avatar}}}</div><# } #><a class="ap-noti-inner" href="{{permalink}}"><# if(ref_type !== \'reputation\'){ #><strong class="ap-not-actor">{{actor}}</strong><# } #> {{verb_label}} <strong class="ap-not-ref">{{ref_title}}</strong><time class="ap-noti-date">{{date}}</time></a></div>',initialize:function(a){this.model=a.model},render:function(){var a=_.template(this.template);return this.$el.html(a(this.model.toJSON())),this}}),AnsPress.views.Notifications=Backbone.View.extend({template:'<button class="ap-droptogg apicon-x"></button><div class="ap-noti-head">{{aplang.notifications}}<# if(total > 0) { #><i class="ap-noti-unseen">{{total}}</i><a href="#" class="ap-btn ap-btn-markall-read ap-btn-small" apajaxbtn apquery="{{JSON.stringify(mark_args)}}">{{aplang.mark_all_seen}}</a><# } #></div><div class="scroll-wrap"></div>',
```

Comment:
Please exclude this file from commit.

Reply:
I will have more time next week to make a cleaner commit, I will exclude this file.

Reply:
Thanks a lot